semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java

```diff
@@ -16,9 +16,696 @@
 
 // DO NOT EDIT, this is an Auto-generated file from
 // buildscripts/semantic-convention/templates/SemanticAttributes.java.j2
+@SuppressWarnings("unused")
 public final class SemanticAttributes {
   /** The URL of the OpenTelemetry schema for these keys and values. */
-  public static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.13.0";
+  public static final String SCHEMA_URL = "https://opentelemetry.io/schemas/1.15.0";
+
+  /**
+   * The type of the exception (its fully-qualified class name, if applicable). The dynamic type of
+   * the exception should be preferred over the static type in languages that support it.
+   */
+  public static final AttributeKey<String> EXCEPTION_TYPE = stringKey("exception.type");
+
+  /** The exception message. */
+  public static final AttributeKey<String> EXCEPTION_MESSAGE = stringKey("exception.message");
+
+  /**
+   * A stacktrace as a string in the natural representation for the language runtime. The
+   * representation is to be determined and documented by each language SIG.
+   */
+  public static final AttributeKey<String> EXCEPTION_STACKTRACE = stringKey("exception.stacktrace");
+
+  /** The name identifies the event. */
+  public static final AttributeKey<String> EVENT_NAME = stringKey("event.name");
```

Comment:
I'm starting to wonder if maybe we should split the files out by type...i.e. event/span/metric

Reply:
Maybe... the counter argument is that attributes may overlap between signals and are expected to have the same semantics. I.e. `MetricAttributes.HTTP_METHOD`, `SpanAttributes.HTTP_METHOD`.

Reply:
What if we could annotate the attributes with some kind of indication on which signals they should be used with?

Reply:
something to think about. I don't want to jump in any direction until things stabilize a bit more, though